ntoskrnl/mm/balance.c

```diff
@@ -272,7 +272,8 @@ MmRebalanceMemoryConsumers(VOID)
 {
     // if (InterlockedCompareExchange(&PageOutThreadActive, 0, 1) == 0)
     {
-        KeSetEvent(&MiBalancerEvent, IO_NO_INCREMENT, FALSE);
+		/* Changed increment from IO_NO_INCREMENT to EVENT_INCREMENT to boost balancer priority */
```

Comment:
That comment should be in the commit log instead.
In the code instead you could just say:
```suggestion
		/* Reset the event and boost balancer priority */
```